fallback.go

```diff
@@ -1,53 +0,0 @@
-package transport
```

Comment:
Removed without replacement (the libp2p constructor will add-in the tcp transport by default if no other transports are specified.